File: Java e JS/historia.java
// Declaração de Variáveis e Tipagem
public class Main {
    public static void main(String[] args) {
    int numero = 10; //Declaração de uma variável inteira
    System.out.println(numero);

// Sintaxe e Estruturas de Controle
    public class Main {
        public static void main(String[] args) {
        System.out.println("Olá, Mundo!"); 
        }
        }

// Funções e Métodos
public class Main {
    public static void saudacao() {
    System.out.println("Olá de uma função em Java!");
    }
    public static void main(String[] args) {
    saudacao(); 
    }
    }

// Orientação a Objetos
public class Pessoa {
    String nome; public Pessoa(String nome) {
    this.nome = nome;
    }
    public void saudacao() {
    System.out.println("Olá, meu nome é " + nome);
    }
    public static void main(String[] args) {
    Pessoa pessoa = new Pessoa("Carlos");
    pessoa.saudacao();
    }
    }

// Execução Assíncrona
public class Main extends Thread {
    public void run() {
    System.out.println("Tarefa emsegundo plano em
    Java!");
    }
    public static void main(String[] args) {
    Main tarefa = new Main();
    tarefa.start();
    }
    }

File: Java e JS/historia.js
// Declaração de Variáveis e Tipagem
let numero = 10;
numero
console.log(numero);
numero = "dez";
string
console.log(numero);

// Sintaxe e Estruturas de Controle
console.log("Olá, Mundo!");

// Funções e Métodos
function saudacao() {
    console.log("Olá de uma função em JavaScript!");
}
saudacao();

// Orientação a Objetos
class Pessoa {
    constructor(nome) {
        this.nome = nome;
    } saudacao() {
        console.log("Olá,meu nome é " + this.nome);
    }
}
const pessoa = new Pessoa("Carlos");
pessoa.saudacao();

// Execução Assíncrona
function tarefa() {
    return new Promise(resolve => {
        setTimeout(() => {
            resolve("Tarefa em segundo plano em JavaScript!");
            }, 1000);
    });
}
async function executarTarefa() {
    const resultado = await tarefa();
    console.log(resultado);
}
executarTarefa();